lighthouse-cli/test/smokehouse/frontends/smokehouse-bin.js

```diff
@@ -13,17 +13,22 @@
 
 /* eslint-disable no-console */
 
-const fs = require('fs');
-const path = require('path');
-const cloneDeep = require('lodash.clonedeep');
-const yargs = require('yargs');
-const log = require('lighthouse-logger');
-const {runSmokehouse} = require('../smokehouse.js');
-const {updateTestDefnFormat} = require('./back-compat-util.js');
-const {LH_ROOT} = require('../../../../root.js');
+import path from 'path';
+import fs from 'fs';
+import url from 'url';
+
+import cloneDeep from 'lodash.clonedeep';
+import yargs from 'yargs';
+// @ts-expect-error: TODO: no types until @types/yargs 16 ... should upgrade yargs first.
```

Comment:
we're using yargs@16.2, so seems fine to `yarn add -D @types/yargs@^16.0.0`? Looks like no type issues. Electron support (or [whatever it's adding](https://github.com/yargs/yargs#simple-example) :) is great, but doesn't seem worth bringing an `any` into the mix to support it otherwise